src/analyzer/protocol/udp/UDP.cc

```diff
@@ -61,7 +62,30 @@ void UDP_Analyzer::DeliverPacket(int len, const u_char* data, bool is_orig,
 
 	int chksum = up->uh_sum;
 
-	if ( ! ignore_checksums && caplen >= len )
+	auto validate_checksum = ! ignore_checksums && caplen >=len;
+	constexpr auto vxlan_len = 8;
+	constexpr auto eth_len = 14;
+
+	if ( validate_checksum &&
```

Comment:
The need for hard-coding VXLAN into UDP is unfortunate, but I don't see another way of doing it either, given the weird checksum semantics.

Reply:
Same feeling, but couldn't think of a less-awkward way to handle it.